src/ProgressOnderwijsUtils.Analyzers/MustUseExpressionResultAnalyzer.cs

```diff
@@ -123,9 +123,10 @@ static bool IsSymbolOfType(ITypeSymbol? symbol, string ns, string name)
             return symbol.Name == name;
         }
 
-        static string Namespace(ITypeSymbol symbol)
+        static string? Namespace(ITypeSymbol symbol)
         {
-            return ContainingNamespaces().Reverse().Aggregate((a, b) => $"{a}.{b}");
+            var parts = ContainingNamespaces().Reverse().ToArray();
+            return parts.Any() ? parts.Aggregate((a, b) => $"{a}.{b}") : default;

```

Comment:
Is het niet makkelijker .Select(x => x.ToString()).JoinStrings(".") te doen?

Reply:
Ik heb geen toegang tot de Utils assembly in deze assembly. Beetje expres, om circulaire dependencies te voorkomen.

Reply:
ah ok, was me niet opgevallen waar we zaten. Alsnog is string.Join en select dan duidelijker toch?

Reply:
Denk het niet, want ik return nu een null bij geen namespace, met die wijziging zou dat dan weer een lege string gaan worden.

Reply:
Ik vraag me af of deze method uebherhaupt moet bestaan, maar zeker zal string.Join efficienter en vast ook korter zijn.